.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -32,47 +36,47 @@ Paste your code here.
 
 </details>
 
-**Logs**
-<details><summary>Terminal output (Screenshots acceptable)</summary>
+
+## Logs
+<details><summary>Terminal output</summary>
 
```

Comment:
**Suggestion** : precise here that logs should be generated using `-v DEBUG` flag

Reply:
Good idea! 4d11443a